components/modal/modal.scss

```diff
@@ -80,41 +80,59 @@
 }
 
 .ui-modal__close-button {
+  align-items: center;
   background: transparent;
   border: 0;
   color: $tx-modal-close-button-color;
   cursor: pointer;
-  line-height: 20px;
+  display: flex;
+  flex-direction: row-reverse;
+  font-size: 13px;
```

Comment:
Would it be possible to use `rem` units? (in this case: `0.8125rem`)

Same for all other `px`... In terms of a11y and responsiveness is better: https://offroadcode.com/rem-calculator/

If you don't want, that's fine too... 👍